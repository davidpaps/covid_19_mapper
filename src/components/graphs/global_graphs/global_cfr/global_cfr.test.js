import * as React from "react";
import { shallow } from "enzyme";
import CFRContainer from "./global_cfr";

describe("Global CFR", () => {
  let wrapper;

  beforeEach(
    () => (wrapper = shallow(<CFRContainer createLineLabels={[]} data={[]} />))
  );

  it("should render a <div />", () => {
    expect(wrapper.find("div").length).toEqual(4);
  });

  it("should render a <Line />", () => {
    expect(wrapper.find("Line").length).toEqual(1);
  });

  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
