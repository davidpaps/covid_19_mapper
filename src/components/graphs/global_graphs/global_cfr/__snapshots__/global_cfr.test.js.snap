// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Global CFR renders correctly 1`] = `ShallowWrapper {}`;
